From 02213311a361ccfbc0e7e12ed9e48017db6c0697 Mon Sep 17 00:00:00 2001
From: Ming Zhou <b42586@freescale.com>
Date: Wed, 16 Jan 2013 16:30:09 +0800
Subject: [PATCH 151/602] ENGR00133115 add 3.0 audio parser for mp3, flac and
 wav formats

modify make file to add the new parser libraries

Signed-off-by: Ming Zhou <b42586@freescale.com>
---
 imx5x/imx5x.mk | 12 +++++++-----
 imx6/imx6.mk   | 11 +++++++----
 2 files changed, 14 insertions(+), 9 deletions(-)

diff --git a/imx5x/imx5x.mk b/imx5x/imx5x.mk
index dd7bd4a..6931e5e 100644
--- a/imx5x/imx5x.mk
+++ b/imx5x/imx5x.mk
@@ -224,11 +224,13 @@ omx_libs := \
 	lib_wb_amr_dec_arm9_elinux			\
 	lib_omx_aac_enc_v2_arm11_elinux \
 	lib_amr_parser_arm11_elinux.3.0 \
-	lib_aac_parser_arm11_elinux.3.0.so \
-	lib_aacd_wrap_arm12_elinux_android.so \
-	lib_mp3d_wrap_arm12_elinux_android.so \
-	lib_vorbisd_wrap_arm11_elinux_android.so \
-
+	lib_aac_parser_arm11_elinux.3.0 \
+	lib_aacd_wrap_arm12_elinux_android \
+	lib_mp3d_wrap_arm12_elinux_android \
+	lib_vorbisd_wrap_arm11_elinux_android \
+	lib_mp3_parser_arm11_elinux.3.0 \
+	lib_flac_parser_arm11_elinux.3.0 \
+	lib_wav_parser_arm11_elinux.3.0 \
 
 
 # Omx excluded libs
diff --git a/imx6/imx6.mk b/imx6/imx6.mk
index c55b5fd..c7a1397 100644
--- a/imx6/imx6.mk
+++ b/imx6/imx6.mk
@@ -228,10 +228,13 @@ omx_libs := \
 	lib_wb_amr_dec_arm9_elinux \
 	lib_omx_aac_enc_v2_arm11_elinux \
 	lib_amr_parser_arm11_elinux.3.0 \
-	lib_aac_parser_arm11_elinux.3.0.so \
-	lib_aacd_wrap_arm12_elinux_android.so \
-	lib_mp3d_wrap_arm12_elinux_android.so \
-	lib_vorbisd_wrap_arm11_elinux_android.so \
+	lib_aac_parser_arm11_elinux.3.0 \
+	lib_aacd_wrap_arm12_elinux_android \
+	lib_mp3d_wrap_arm12_elinux_android \
+	lib_vorbisd_wrap_arm11_elinux_android \
+	lib_mp3_parser_arm11_elinux.3.0 \
+	lib_flac_parser_arm11_elinux.3.0 \
+	lib_wav_parser_arm11_elinux.3.0 \
 
 
 
-- 
1.8.0

